keras_cv/custom_ops/__init__.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2022 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
```

Comment:
I see a `_pariwise_iou_op.so` being included,
1) should we just have a single `custom_op.so` that includes all the targets?
2) if a user wants to change some ops implementation, maybe we should have a documentation on how to do that?

Reply:
yeah please do so, my main ask here is how users can build and update the *.so file once they change the implementation.
having a single .so object seems better to me, even tensorflow itself only contains very few .so.

Reply:
Gotcha -- that is already covered in `Tests that require custom ops` in `CONTRIBUTING.md`. I've updated that file to refer to the instructions for building  the binaries in the `Contributing Custom Ops` section

I've also renamed the .so to `_keras_cv_custom_ops.so` to indicate that later ops should be included in this same binary.

Reply:
Sounds good!